bin/failsafe

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env node
 
-require('../lib/index')
+require('../lib/src/index')
```

Comment:
This was cased by `import * as packageJson from '../package.json';` in Failsafe.ts and the `resolveJsonModule` in tsconfig.json